cloud_functions/test/flagCryptominer-error-tests.js

```diff
@@ -0,0 +1,93 @@
+// These tests cover various error cases for the flagCryptominer cloud function. Since everything
+// about this operates in the background, we want to make sure that errors
+// Based on https://cloud.google.com/functions/docs/testing/test-background#unit_tests
```

Comment:
"make sure that errors" ... trails off

Reply:
https://media.giphy.com/media/xB7FIRaxrtNaE/giphy.gif

Sorry. Got lost mid thought. Will fix.

Reply:
I think I got interrupted in the middle of rewriting this comment block. I completed that rewrite (rather than just completing the sentence). My aim is to comment first on what, then why, then implementation details.